src/video.tsx

```diff
@@ -193,6 +199,10 @@ const Video = React.forwardRef<HTMLVideoElement, VideoProps>(
                 style={{
                     // For some reason adding this style with classnames doesn't add the border radius in safari
                     borderRadius: !borderRadiusContext ? 0 : vars.borderRadii.container,
+                    visibility: showPoster ? 'hidden' : 'visible',
+                    position: showPoster || ratio !== 0 ? 'absolute' : 'static',
+                    width: showPoster ? posterWidth : '100%',
+                    height: showPoster ? posterHeight : '100%',
```

Comment:
In the case of `aspectRatio={0}`, if poster image and video have different aspect ratios, there will be a weird jump. But I think there is no way to avoid that

Reply:
This is expected behaviour, and it's how video with poster works in html